src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -438,34 +466,105 @@ static void MX_GPIO_Init(void)
 
     /* GPIO Ports Clock Enable */
     __HAL_RCC_GPIOC_CLK_ENABLE();
+    __HAL_RCC_GPIOF_CLK_ENABLE();
     __HAL_RCC_GPIOH_CLK_ENABLE();
     __HAL_RCC_GPIOA_CLK_ENABLE();
     __HAL_RCC_GPIOB_CLK_ENABLE();
     __HAL_RCC_GPIOD_CLK_ENABLE();
     __HAL_RCC_GPIOG_CLK_ENABLE();
 
     /*Configure GPIO pin Output Level */
-    HAL_GPIO_WritePin(GPIOB,
-                      motor_0_esf_Pin | LD3_Pin | motor_0_reset_Pin | motor_0_coast_Pin |
-                          motor_0_mode_Pin | LD2_Pin | motor_0_direction_Pin |
-                          motor_0_brake_Pin,
+    HAL_GPIO_WritePin(
+        GPIOF,
+        wheel_motor_back_right_esf_Pin | wheel_motor_front_right_reset_Pin |
+            wheel_motor_front_right_coast_Pin | wheel_motor_front_right_mode_Pin |
+            wheel_motor_front_right_direction_Pin | wheel_motor_front_right_brake_Pin |
+            wheel_motor_front_right_esf_Pin,
+        GPIO_PIN_RESET);
+
+    /*Configure GPIO pin Output Level */
+    HAL_GPIO_WritePin(
+        GPIOC, wheel_motor_back_right_coast_Pin | wheel_motor_back_left_direction_Pin,
+        GPIO_PIN_RESET);
+
+    /*Configure GPIO pin Output Level */
+    HAL_GPIO_WritePin(GPIOA,
+                      wheel_motor_back_right_brake_Pin |
+                          wheel_motor_back_right_reset_Pin |
+                          wheel_motor_back_right_direction_Pin,
```

Comment:
Why does the GPIO write for front_right have so many more pins than back_right?

Reply:
It doesn't, it just looks like it here because different pins fall under different GPIO "groups".